lib/backend/compat/compat.go

```diff
@@ -732,7 +732,7 @@ func toDatastoreGlobalBGPConfigKey(key model.GlobalBGPConfigKey) model.GlobalBGP
 }
 
 // toDatastoreGlobalBGPConfigList modifies the Global BGP Config List interface to the one required by
-// the datastore (for back-compatibility with what is expected in teh etcdv2 datastore driver).
+// the datastore (for back-compatibility with what is expected in teh etcdv3 datastore driver).
```

Comment:
spotted an old typo: `teh`

Reply:
yeah saw that and thought meh.  I've actually just reverted that file.  We can probably bin the combat module completely - but wanted to keep it around until I'd done the migration code.